src/darwin/Framework/CHIPTests/MTRCertificateTests.m

```diff
@@ -126,10 +126,16 @@ - (void)testGenerateIntermediateCert
 
     __auto_type * intermediateKeys = [[MTRTestKeys alloc] init];
     XCTAssertNotNil(intermediateKeys);
+    // TODO:  there are several places in our tests where we call `copyPublicKey` on objects without first
```

Comment:
Yes, but we know we're using MTRTestKeys here, and that it implements copyPublicKey.  I don't think we need this comment.

Reply:
my ambition was to make "there's no implementation of `copyPublicKey`" a compile-time error, but it looks like if it's optional in the protocol, that's not going to happen, so agreed and i'll take the comment out.